client/src/views/Models/components/DrilldownPanel/MetadataPane.vue

```diff
@@ -1,144 +1,110 @@
-              {{value}}
-            </div>
-          </div>
-        </div>
-      </div>
-    </collapsible-item>
-
-    <collapsible-item slot="item" class="flex-grow-1">
-      <div slot="title">Text Snippets</div>
-      <div slot="content" class="h-100 position-absolute">
-        <div v-for="(item, index) in getKnowledge" :key="index" class="snippet-container" @click="showMoreHandler(item.doi)">
-          <div class="snippet-title">
-            <a target="_blank" :href="item.URL">{{item.title}}</a>
-          </div>
-          <span v-for="(snippet, index) in item.highlight" :key="index" v-html="snippet" class="snippet-highlights"/>
-        </div>
-      </div>
-    </collapsible-item>
-
-    <div slot="empty" class="alert alert-info" role="alert">
+  <div class="metadata-list">
+    <div v-if="isEmptyMetadata" class="alert alert-info">
       No metadata at the moment
     </div>
-  </collapsible-container>
+
+    <details
+      class="metadata" open
+      v-else
+      v-for="(datum, index) in metadata" :key="index"
+    >
+      <template v-if="isTypeCodeSpanReference(datum)">
+        <summary :title="datum.uid">Code Reference</summary>
+        <div class="metadata-content">
+          <h6>Type</h6>
+          <p>{{ datum.code_type | capitalize-formatter }}</p>
+          <h6>File</h6>
+          <p>
+            <a :href="datum.file_id">{{ datum.file_id }}</a><br>
+            {{ sourceFilePosition(datum) }}
+          </p>
+        </div>
+      </template>
+
+      <aside class="metadata-provenance">
+        {{ datum.provenance.method }}
+        <time :datetime="datum.provenance.timestamp">
+          {{ provenanceDate(datum.provenance.timestamp) }}
+        </time>
+      </aside>
+    </details>
+  </div>
 </template>
 
 <script lang="ts">
-  import _ from 'lodash';
-
-  import Component from 'vue-class-component';
   import Vue from 'vue';
+  import Component from 'vue-class-component';
   import { Prop } from 'vue-property-decorator';
 
-  import CollapsibleContainer from '@/components/Collapsible/CollapsibleContainer.vue';
-  import CollapsibleItem from '@/components/Collapsible/CollapsibleItem.vue';
-  import { CosmosTextSnippet } from '@/types/typesCosmos';
+  import * as GroMET from '@/types/typesGroMEt';
+  import { formatFullDateTime } from '@/utils/DateTimeUtil';
 
-  const components = {
-    CollapsibleContainer,
-    CollapsibleItem,
+  const defaultCodeSpanReference: any = {
+    code_type: GroMET.CodeType.Identifier,
+    file_id: null,
+    line_begin: null,
+    line_end: null,
+    col_begin: null,
+    col_end: null,
   };
 
-  @Component({ components })
+  @Component
   export default class MetadataPane extends Vue {
-    @Prop({ default: null }) data: any;
-
-    showModal: boolean = false;
-    dataLoading = false;
+    @Prop({ default: null }) data: any[];
+
+    /** Clean the metadata to match our expected format.
+     * TODO: This is a temporary solution and should be replaced with
```

Comment:
Not sure what you mean by this comment. I don't think we will be getting metadata for nodes from donu, we will still be reading metadata from GroMEts.

Reply:
From my understanding, we are using GroMET files in the front-end because DONU API will not be finished before the August demo. Isn't the final goal of HMI to not be aware of GroMET at all?
@adamocarolli, is what I'm saying accurate, or I'm just out in the boonies?

Reply:
You're right, we are not working GroMEts directly, however, the metadata embedded in them I don't think it will change that much.